components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -294,9 +294,9 @@ public DataSection genesBody(Patient patient) throws Exception
 
     public void variantsSetup(Set<String> enabledFields) throws Exception
     {
-        String sectionName = "genes";
-        Set<String> present = this.enabledHeaderIdsBySection.get(sectionName);
-        if (present == null || present.isEmpty()) {
+        String sectionName = "variants";
```

Comment:
Excel export never includes variants with this change. `sectionName` must also be changed in the next two methods, `variantsHeader` and `variantsBody`,